lib/test.js

```diff
@@ -108,6 +109,12 @@ Test.prototype.test = function (name, opts, cb) {
 
 Test.prototype.comment = function (msg) {
     var that = this;
+
+    // Previous behavior involved `toString` calls on objects, i.e. emitting `[object Object]`.
```

Comment:
sadly I still don't think this is sufficient - previously, multiple arguments were ignored, and I still might have done `arr.forEach(t.comment)` for example, and relied on the stringification despite passing 3 arguments. I think that the first argument must never be stringified to avoid a breaking change.